tools/mmp/driver.cs

```diff
@@ -110,6 +110,7 @@ public static partial class Driver {
 
 		static bool is_extension;
 		static bool frameworks_copied_to_bundle_dir;	// Have we copied any frameworks to Foo.app/Contents/Frameworks?
+		static bool just_run_registrar;
```

Comment:
Just for the same of making the code more similar to mtouch IMHO it would be nicer to copy that code (i.e. have an `Action` enum (with just two entries in this case: Build and RunRegistrar) and an `action` variable that specifies what to do).